types/utility_descriptions.h

```diff
@@ -39,11 +39,11 @@ struct BridgedValueDescription: public ValueDescription {
     virtual void * getArrayElement(void * val, uint32_t element) const override;
     virtual const void * getArrayElement(const void * val, uint32_t element) const override;
     virtual const ValueDescription &
-    getArrayElementDescription(const void * val, uint32_t element) const;
+        getArrayElementDescription(const void * val, uint32_t element) const override;
     virtual void setArrayLength(void * val, size_t newLength) const override;
     virtual size_t getTupleLength() const override;
-    virtual std::vector<std::shared_ptr<const ValueDescription> >
-    getTupleElementDescriptions() const override;
+    virtual std::vector<std::shared_ptr<const ValueDescription>>
```

Comment:
Oh? I thought it was an old habit from when compilers confused it with `operator >>`